### PR TITLE
Revert "Removes NCR MP + Sergeant MP loadout" and gives them razors

### PR DESCRIPTION
**Pull request by companoo**
someone got salty MP's were enforcing uniform regulations

+ adds razors there loadouts
+ makes MP's corporals and removes reference to the SEA in there desc

### CODE REVIEW

**Comment by ErwinSnowden** [comment]
speedmerge

**Comment by HanSolo1519** [comment]
sneedmerge

**Comment by BadAtThisGame302** [comment]
_**speedmerge**_

File: code/modules/jobs/job_types/ncr.dm [hunk 1]
```diff
@@ -827,8 +840,8 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
 /datum/job/ncr/f13combatmedic
 	title = "NCR Combat Medic"
 	flag = F13COMBATMEDIC
-	total_positions = 2
-	spawn_positions = 2
+	total_positions = 1
```
Comment (RonBag) [inline code/modules/jobs/job_types/ncr.dm:843]: Undocumented change.

**Review by RonBag — CHANGES_REQUESTED** [review]

**Comment by companoo** [comment]
shouild be fixed

**Review by RonBag — APPROVED** [review]
The razor is a little memey, but looks good

File: code/modules/jobs/job_types/ncr.dm [hunk 2]
```diff
@@ -829,7 +842,7 @@ Weapons		Service Rifle, Grease Gun, 9mm pistol, all good.
 	flag = F13COMBATMEDIC
 	total_positions = 2
 	spawn_positions = 2
-	description = "You are a senior enlisted with a medical skill set. You work closely with your squad, taking orders from your officers. You have the authority to command troopers if there are no non-commissioned officers present."
+	description = "Your enlisted with a medical skill set. You work closely with your squad, taking orders from your officers. You have the authority to command troopers if there are no non-commissioned officers present."
```
Comment (BadAtThisGame302) [inline code/modules/jobs/job_types/ncr.dm:845]: You're an*

Comment (companoo) [reply code/modules/jobs/job_types/ncr.dm]: whoops thank

**Comment by Sinbest** [comment]
speedmerge

**Pull request merged**